Linux metadata install path issue

Linux install fails with error

```
-- Installing: /usr/local/share/metainfo/org.zdoom.GZDoom.metainfo.xml
CMake Error at src/cmake_install.cmake:61 (file):
  file INSTALL cannot find
  "/home/marcus/Code/repos/gzdoom/src/posix/org.zdoom.GZDoom.svg": No such
  file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:116 (include)


make: *** [Makefile:100: install] Error 1
```

---

I left a review on #3188 last week, but it was left on "Pending". I thought that meant it was waiting for approval, but I'm now thinking I did something wrong. How should I go about leaving feedback in the future?

<img width="929" height="807" alt="Image" src="https://github.com/user-attachments/assets/1c7f906e-cb8d-4a31-8df3-02af20427236" />

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1461,7 +1461,7 @@ if( UNIX AND NOT APPLE )
 	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/posix/freedesktop/org.zdoom.GZDoom.metainfo.xml
 			DESTINATION share/metainfo
 			PERMISSIONS WORLD_READ GROUP_READ GROUP_EXECUTE OWNER_READ OWNER_WRITE)
-	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/posix/org.zdoom.GZDoom.svg
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/posix/freedesktop/org.zdoom.GZDoom.svg
 			DESTINATION share/icons/hicolor/scalable/apps
 			PERMISSIONS WORLD_READ GROUP_READ GROUP_EXECUTE OWNER_READ OWNER_WRITE)
 endif()

